src/graphnet/models/graphs/graphs.py

```diff
@@ -54,3 +54,41 @@ def __init__(
             perturbation_dict=perturbation_dict,
             seed=seed,
         )
+
+
+class IsolatedNodes(GraphDefinition):
```

Comment:
Maybe we could rename this from `IsolatedNodes`  to `EdgelessGraph`, or something else that indicates exactly how this representation is different from existing representations?